activesupport/test/cache/stores/redis_cache_store_test.rb

```diff
@@ -208,13 +227,30 @@ def test_increment_expires_in
       end
 
       # key exist but not have expire
-      redis_backend { |r| r.set "#{@namespace}:dar", 10 }
-      @cache.increment "dar", 1, expires_in: 60
-      redis_backend do |r|
+      redis_backend(@cache_no_ttl) { |r| r.set "#{@namespace}:dar", 10 }
+      @cache_no_ttl.increment "dar", 1, expires_in: 60
+      redis_backend(@cache_no_ttl) do |r|
```

Comment:
This test (and the mirrored "`#decrement` test) were updated to use `@cache_no_ttl` because `@cache` sets a default TTL of `60`, which would cause this test to pass, even if `expires_in` option was removed from the `#increment` call.